src/main/webapp/app/scores/exercise-scores.component.ts

```diff
@@ -71,26 +84,28 @@ export class ExerciseScoresComponent implements OnInit, OnDestroy {
     }
 
     getResults() {
```

Comment:
Would be nice to add JSDoc to all public methods that you changed, since we want to have all code documented

Reply:
I think it will be also better to specify here explicitly the returned type of the method

Reply:
This PR is just a bugfix, I won't add missing documentation here.